examples/igrantio/test-entrypoint.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set +e
+
+docker-compose -f docker-compose.yaml up mongo postgresql keycloak api caddy -d
```

Comment:
Can you change it to:
`docker-compose -f docker-compose.yaml up -d mongo postgresql keycloak api caddy
`
The flag should be before the services names if you want to run this command locally.